Lib/fontTools/feaLib/ast.py

```diff
@@ -319,21 +319,20 @@ def addDefinition(self, definition):
         assert isinstance(definition, MarkClassDefinition)
         self.definitions.append(definition)
         for glyph in definition.glyphSet():
-            if glyph in self.definitions:
-                otherLoc = self.definitions[glyph].location
-                raise FeatureLibError(
-                    "Glyph %s already defined at %s:%d:%d" % (
-                        glyph, otherLoc[0], otherLoc[1], otherLoc[2]),
-                    definition.location)
+# Fixing this code fails tests!
```

Comment:
No idea. Want to find out? I don't really care. I didn't write the code. I don't mind which way it jumps.